.env.template

```diff
@@ -1,6 +1,13 @@
-THOTH_CEPH_BUCKET=
-THOTH_CEPH_BUCKET_PREFIX=
+# source ceph
+THOTH_DEPLOYMENT_NAME="ocp-test"
+
+THOTH_S3_ENDPOINT_URL="https://s3.upshift.redhat.com/
+THOTH_CEPH_BUCKET="thoth"
+THOTH_CEPH_BUCKET_PREFIX="data"
 THOTH_CEPH_KEY_ID=
 THOTH_CEPH_SECRET_KEY=
-THOTH_DEPLOYMENT_NAME=
-THOTH_S3_ENDPOINT_URL=
+
+# dest ceph
+THOTH_DOCUMENT_SYNC_DST="s3://thoth/data/test"
```

Comment:
the `test` subprefix is for cleanup ? -> maybe we should use `s3://thoth/test` to avoid messing with other data + add a note for testers to use their own subprefix under `test` if they want ?

Reply:
ah right, will update it.